src/middlewared/middlewared/job.py

```diff
@@ -291,7 +289,7 @@ def __init__(self, middleware, method_name, serviceobj, method, args, options, p
         self.app = app
         self.audit_callback = audit_callback
 
-        self.id = None
+        self.id = int(uuid4())
```

Comment:
You can do `self.id = uuid4().int`

Reply:
@undsoft will such long ints work well with JSON?
```
>>> uuid4().int
311364041177499672922985169261652976368
>>> uuid4().int
262998890057434762054815435825767399766
>>> uuid4().int
106487518651827065074465048916488123677
```

Reply:
I was hemming and hawing about that one. I may make it a string rather than number (e.g. `"106487518651827065074465048916488123677"`)  to make random JSON libraries happier. UI seems to not worry about consuming UUIDs here (nothing broke in an obvious way).